core/drivers/tzc380.c

```diff
@@ -92,6 +99,38 @@ void tzc_init(vaddr_t base)
 			   BUILD_CONFIG_NR_MASK) + 1;
 }
 
+void tzc_security_inversion_en(vaddr_t base)
+{
+	write32(1, base + SECURITY_INV_EN_OFF);
+}
+
+void tzc_enable_region(uint8_t region)
```

Comment:
`tzc_region_enable()` would be more consistent with for instance `tzc_security_inversion_en()` and `tzc_int_clear()`

Reply:
It wouldn't hurt with a comment with a short description of some of the more important functions